bin/stats.py

```diff
@@ -48,8 +49,8 @@ def problem_stats(problems):
         # Roughly in order of importance
         ("  time", lambda p: p.limits.time_limit, 0),
         ("yaml", "problem.yaml"),
-        ("tex", "problem_statement/problem*.tex", 1),
-        ("sol", "problem_statement/solution*.tex", 1),
+        ("tex", str(latex.PdfType.PROBLEM.path(ext="*.tex")), 1),
```

Comment:
This now no longer matches `problem-slide.*.tex`, but that's probably fine?

I also don't think we need to add the problem slide as separate item to `bt stats`, because not many people use this, I think.